openedx/features/content_tagging/rules.py

```diff
@@ -1,24 +1,25 @@
 """Django rules-based permissions for tagging"""
 
+from typing import List
+
 import openedx_tagging.core.tagging.rules as oel_tagging
 import rules
 from django.contrib.auth import get_user_model
 
 from common.djangoapps.student.auth import is_content_creator
+from organizations.models import Organization
 
 from .models import TaxonomyOrg
 
 User = get_user_model()
 
 
-def is_taxonomy_admin(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool:
+def is_taxonomy_user(user: User, taxonomy: oel_tagging.Taxonomy) -> bool:
```

Comment:
This method does sometimes take `taxonomy=None` from the checks below, so could you please re-add that?
```suggestion
def is_taxonomy_user(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool:
```